stablehlo/reference/ProcessGrid.h

```diff
@@ -212,21 +255,44 @@ class ProcessGrid {
                                                      ProcessId processId,
                                                      const Tensor &operand);
 
+  /// Sends `inputs` to a channel with `channelId`.
+  /// The channel with `channel_id` is emptied before the receiving process can
+  /// receive values. If there are multiple processes sending data to a
+  /// duplciate `channelId`, the behavior is undefined.
+  void send(ArrayRef<Tensor> inputs, ChannelId channelId, ProcessId processId);
+
  private:
   /// StableHLO `num_replicas`.
   const uint32_t numReplicas_;
 
   /// StableHLO `num_partitions`.
   const uint32_t numPartitions_;
 
-  /// Interal queue of strings which represents `func::FuncOp` mnemonic that
+  /// Internal queue of strings which represents `func::FuncOp` mnemonic that
   /// returns a vector of Tensor. The function name is stored instead of the
   /// vector of tensors to save memory. See `ThreadSafeQueue`.
   detail::ThreadSafeQueue<StringAttr> infeed_;
 
   /// See `ThreadSafeQueue`.
   detail::ThreadSafeQueue<SmallVector<Tensor>> outfeed_;
```

Comment:
Could you also add a description for outfeed here? All the other member variables have great descriptions of how they're used